data/defaults.yaml

```diff
@@ -0,0 +1 @@
+puppetboard::manage_selinux: "%{facts.os.selinux.enabled}"
```

Comment:
can you keep that in init.pp? that allows puppet-strings to pick up the default and write it to REFERENCE.md